roles/os_zabbix/library/zbxapi.py

```diff
@@ -69,23 +84,25 @@ class ZabbixAPI(object):
         'Usermedia': ['get'],
     }
 
-    def __init__(self, data={}):
-        self.server = data['server'] or None
-        self.username = data['user'] or None
-        self.password = data['password'] or None
-        if any(map(lambda value: value == None, [self.server, self.username, self.password])):
+    def __init__(self, data=None):
+        if not data:
+            data = {}
```

Comment:
This is a nit, but I like how the canonical example for pylint does this:

```
def my_method(self, arg1, extras=None):
    extras = extras or []
```

Consider changing this to simply:
```
data = data or {}
```

From:
http://nedbatchelder.com/blog/200806/pylint.html